x-pack/packages/security-solution/side_nav/src/solution_side_nav_panel.tsx

```diff
@@ -354,7 +355,11 @@ interface SolutionSideNavPanelItemProps {
 const SolutionSideNavPanelItem: React.FC<SolutionSideNavPanelItemProps> = React.memo(
   function SolutionSideNavPanelItem({ item, onClose }) {
     const { tracker } = useTelemetryContext();
-    const panelLinkClassNames = classNames('solutionSideNavPanelLink');
+    const { euiTheme } = useEuiTheme();
+    const panelLinkClassNames = classNames(
+      'solutionSideNavPanelLink',
```

Comment:
I think the `solutionSideNavPanelLink` class is no longer required, can we remove it?

Reply:
Addressed in https://github.com/elastic/kibana/pull/201823/commits/9e740215c2135e68af607cc9cbab4ebb8739ded6